test/test.rethrow.js

```diff
@@ -1,23 +1,78 @@
+var path = require('path');
+
+delete require.cache[path.resolve(path.join(__dirname, '../twig.js'))];
+
 var Twig = Twig || require("../twig"),
     twig = twig || Twig.twig;
 
-describe("Twig.js Rethrow ->", function() {
-    it("should throw a \"Unable to parse 'missing'\" exception", function() {
-        (function(){
+describe("Twig.js Rethrow ->", function () {
```

Comment:
I think the rest of our tests use `function()` rather than `function ()`. Can you update accordingly?